tensorflow/core/kernels/mkl/mkl_batch_matmul_op.cc

```diff
@@ -242,12 +242,12 @@ class BatchMatMulMkl : public OpKernel {
                               .TypeConstraint<TYPE>("T")                      \
                               .Label(mkl_op_registry::kMklNameChangeOpLabel), \
                           BatchMatMulMkl<CPUDevice, TYPE, true>)
-#ifdef ENABLE_MKL
+#ifdef INTEL_MKL
```

Comment:
Please remove this. The whole file is already under `#ifdef INTEL_MKL` (see line 26).